src/NewTools-Debugger/StDebuggerConfigurationToolbarCommandTreeBuilder.class.st

```diff
@@ -0,0 +1,46 @@
+Class {
+	#name : #StDebuggerConfigurationToolbarCommandTreeBuilder,
+	#superclass : #StDebuggerCommandTreeBuilder,
+	#category : #'NewTools-Debugger-Model'
+}
+
+{ #category : #accessing }
+StDebuggerConfigurationToolbarCommandTreeBuilder class >> displayStrategy [
+	^CmUIDisplayAsGroup
+]
+
+{ #category : #accessing }
+StDebuggerConfigurationToolbarCommandTreeBuilder class >> groupDescription [
+	^ 'Debugger configuration actions'
+]
+
+{ #category : #accessing }
+StDebuggerConfigurationToolbarCommandTreeBuilder class >> groupName [ 	
+	^'Configuration'
+]
+
+{ #category : #building }
+StDebuggerConfigurationToolbarCommandTreeBuilder >> buildDebuggerCommandGroup [
+
+	| group |
+	self group.
```

Comment:
this line looks useless (same as line 27 without assignment)

Reply:
I do not have this in my image, actually that class was removed I think.
I will check.

Reply:
Yes, it was removed by https://github.com/pharo-spec/NewTools/pull/467/commits/9a9e2564ea92afd24be6e8dd044d111e6505ff59